userguide/system.rst

```diff
@@ -2333,11 +2333,11 @@ complete these fields:
   issue that describes the problem, and if applicable, what steps can
   be taken to reproduce it.
 
-After completing the fields, click the :guilabel:`SUBMIT` button to
-automatically generate and upload the report to
-https://redmine.ixsystems.com/projects/freenas/issues. A pop-up menu
-provides a URL so to view status or add additional information to the
-report.
+Click :guilabel:`SUBMIT` to automatically generate and upload the report
+to https://redmine.ixsystems.com/projects/freenas/issues. This process
+can take several minutes, depending on the debug file size. A pop-up
```

Comment:
Avoid the halting sentence and giving the user information that doesn't help them:
```suggestion
can take several minutes while information is collected and sent.
```